constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -1471,6 +1763,28 @@ protected static Task createTaskResponse(String string) {
         return new Gson().fromJson(transformed, Task.class);
     }
 
+    /**
+     * Transforms a JSON string to a new JSON string for easy Gson parsing into an Items response.
+     * Using JSON objects to acheive this is considerably less error prone than attempting to do it in
+     * a Gson Type Adapter.
+     */
+    protected static ItemsResponse createItemsResponse(String string) {
+        JSONObject json = new JSONObject(string);
+        String transformed = json.toString();
+        return new Gson().fromJson(transformed, ItemsResponse.class);
+    }
+
+    /**
+     * Transforms a JSON string to a new JSON string for easy Gson parsing into an Variations response.
+     * Using JSON objects to acheive this is considerably less error prone than attempting to do it in
+     * a Gson Type Adapter.
+     */
+    protected static VariationsResponse createVariationsResponse(String string) {
+        JSONObject json = new JSONObject(string);
+        String transformed = json.toString();
+        return new Gson().fromJson(transformed, VariationsResponse.class);
```

Comment:
I don't think this will fully serialize the Variation response since ConstructorVariation object doesn't match the response 1:1. Probably will need a custom serializer. Maybe I am missing where this is handled.

Same for itmes response.